buildh/lipids.py

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pathlib
+import json
+
+def read_lipids_topH(filenames):
+    """Generate a list of lipid hydrogen topologies.
+
+    This function read a list of json files containing
+    the topology of a united-atom lipid for reconstructing
+    the missing hydrogens.
+
+    The list topologies is stored as dictionary were the key is
+    "ForceField_lipidname" (e.g Berger_POPC, CHARMM_DMPC)
+
+    Parameters
+    ----------
+    filenames : list of str
+        List of json files containing topologies.
+
+    Returns
+    -------
+    dictionary of dictionary
+        the lipid topologies.
+
+    Raises
+    ------
+    UserWarning
+        When a file doesn't have a correct format.
+    """
+
+    lipids_topH = {}
```

Comment:
Why do you call it `lipids_topH` ? I don't understand the `H`.

Reply:
for hydrogens :)  topH = topology for hydrogens

Reply:
Yes, that's what I guessed, but when I read `lipids_topH` I imagine there are Hs there. To understand "for" in "topology for hydrogens", I'll call it `lipids_top4H`. Maybe we can call it merely `lipid_top` (without s because there's only one lipid in the json file)? Sorry if I'm too picky 😉

Reply:
I would keep the 's' because it is a dictionary which contains all supported lipids topologies.
But yes i can call it "lipids_top4H" :)

Reply:
or "lipids_tops"